bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/NetworkRequestInfo.kt

```diff
@@ -0,0 +1,5 @@
+package com.bugsnag.android.performance
+
+class NetworkRequestInfo(
+    var url: String? = null
```

Comment:
formatting:

```suggestion
    var url: String? = null,
```

not sure why the linter / formatter didn't pick that up 🤔